src/winrt-component-full/winrt-component-full.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net9.0-windows10.0.22621.0</TargetFramework>
+    <Platforms>x64;arm64</Platforms>
+    <Nullable>enable</Nullable>
+    <AssemblyName>winrt_component_full</AssemblyName>
+
+    <!-- Enable building with 'dotnet msbuild' -->
+    <EnableMsixTooling>true</EnableMsixTooling>
+      
+    <!-- CsWinRT settings -->
+    <WindowsSdkPackageVersion>10.0.22621.35-preview</WindowsSdkPackageVersion>
+    <CsWinRTComponent>true</CsWinRTComponent>
+
+    <!-- Size saving settings -->
+    <PublishAot>true</PublishAot>
+    <NativeLib>Shared</NativeLib>
+    <InvariantGlobalization>true</InvariantGlobalization>
+    <OptimizationPreference>Size</OptimizationPreference>
+    <StackTraceSupport>false</StackTraceSupport>
+    <UseSystemResourceKeys>true</UseSystemResourceKeys>
```

Comment:
Although, I have a question - I didn't actually look at this before I originally merged it. Why do we measure the minimal possible thing? Wouldn't it be more representative of real world to drop these? (The other rt-sz benchmarks are also not measuring the minimal possible thing)

Reply:
Good point. I can remove them here and just leave them on the minimal sample. For that I kinda just want to monitor how small you can make a minimal WinRT component, where you'd always want to optimize for size, as you're going to be consumed by some native application, and including unnecessary .NET feature you know your WinRT component is not going to need anyway would just be wasteful otherwise.